tests/rules/javascript/test.event_listener_fourth.js

```diff
@@ -0,0 +1,128 @@
+import JavaScriptScanner from 'scanners/javascript';
+import { VALIDATION_WARNING } from 'const';
+import { singleLineString } from 'utils';
+import * as messages from 'messages';
+
+describe('event_listener_fourth', () => {
+  it('should not allow a true literal', () => {
+    var code = 'window.addEventListener("click", function(){}, false, true);';
```

Comment:
Nice comprehensive tests.

I wonder if we could add something to someone using `window.addEventListener.call` and `window.addEventListener.apply`. Presumably the old code doesn't go that far?